metricbeat/etc/fields.yml

```diff
@@ -8,11 +8,83 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["common", "Common fields"]
   - ["redis-info", "Redis Info Stats"]
+  - ["mysql-status", "MySQL Status"]
+
+common:
+  type: group
+  description: >
+    Contains common fields available in all event types.
+  fields:
+    - name: beat.name
+      description: >
+        The name of the Beat sending the log messages. If the shipper name is set
+        in the configuration file, then that value is used. If it is not set,
+        the hostname is used.
+
+    - name: beat.hostname
+      description: >
+        The hostname as returned by the operating system on which the Beat is
+        running.
+
+    - name: "@timestamp"
+      type: date
+      required: true
+      format: YYYY-MM-DDTHH:MM:SS.milliZ
+      example: 2015-01-24T14:06:05.071Z
+      description: >
+        The timestamp when the log line was read. The precision is in
+        milliseconds. The timezone is UTC.
+
+    - name: response_time
+      type: long
+      required: true
+      description: >
+        Event Reponse time in nano seconds
+mysql-status:
+  type: group
+  description: >
+    MySQL Stats
+  fields:
+    - name: mysql-status
```

Comment:
@andrewkroh @monicasarbu I looked at the mysql-status thing and I actually remember (could be wrong) to call the namespaces for each metricset `module-metricset`. See implementation here: https://github.com/elastic/beats/blob/master/metricbeat/helper/metricset.go#L94 I quite like the dash instead of the _ here as it is somehow special. Question: Should we keep it that way or change for all?

Second question that this brings up is about Aborted_clients and similar things. What would we do if we get from a system `Aborted-Clients` or `aborted clients`. Do we use a normalising function? Perhaps we should discuss this in a separate issue?